### DISCUSSION TITLE
add pentium based rise mp6 processors and dm&p vortex86

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#4471] [Ideas]
even if they are pentium based i want to see them

frequencies
mp6
166mhz
190mhz
200mhz
240mhz
250mhz
vortex86
same as mp6

**Comment by TC1995** [comment]
For now, just use the current ones.

**Comment by Torinde** [comment]
FYI:
Vortex86 original is the same as SiS55x - a combination of Rise mP6 + Mirage 315 2D + SiS540 chipset

Vortex86SX is not related to Vortex86/mP6 at all. It's using totally different CPU from RDC and its instruction set is 486SX
Vortex86DX/MX/EX and further - additions to Vortex86SX

**Reply by Torinde** [reply]
There is one last mP6 CPU: STMicro [STPC Vega](https://theretroweb.com/chip/documentation/stpc-vega-636e71d164d5d242160126.pdf)

**Comment by ghost** [comment]
ok
